lib/src/client.dart

```diff
@@ -357,3 +352,9 @@ class RemoteAction<O, S> {
     }
   }
 }
+
+class GenkitStream<S, F> extends StreamView<S> {
```

Comment:
For users who do not care about the `Stream` and just want the result, you could also have a convenience getter `Future<F> get onResult`, which drains the stream and returns the result, throwing like a normal future would.